shared-lib/spiutils/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+
+name = "spiutils"
+version = "0.1.0"
+authors = [ "Oskar Senft <osk@google.com>" ]
+license = "MIT"
+keywords = ["spi-utils"]
```

Comment:
`spi-utils` isn't really a good keyword. I don't expect this to be uploaded to crates.io (do you?) so I don't think we need to specify keywords.

Reply:
Removed.